### PR TITLE
#1608 app:update command documented

### PR DESCRIPTION
**Pull request by bluszcz**
* app:update command
* unified lowercase

### CODE REVIEW

File: admin_manual/configuration_server/occ_command.rst [hunk 1]
```diff
@@ -242,6 +243,14 @@ You can get the full filepath to an app::
 
 .. _background_jobs_selector_label:
```
Comment (skjnldsv) [inline admin_manual/configuration_server/occ_command.rst]: This should stay just before the `Background jobs selector` title :)

Comment (bluszcz) [reply admin_manual/configuration_server/occ_command.rst]: outch 🦌

**Comment by bluszcz** [comment]
> see comment
I think I fixed it...

**Comment by skjnldsv** [comment]
@wiswedel and 18 :p

/backport to stable18

**Comment by skjnldsv** [comment]
Ah right, it was done long time ago on 18 already :D

**Comment by wiswedel** [comment]
> @wiswedel and 18 :p

That already happened in the first place. Only 16 and 17 were missing.

**Comment by wiswedel** [comment]
ref https://github.com/nextcloud/server/pull/11053

**Pull request merged**